src/js/dropdown-menu/dropdown-menu-view.js

```diff
@@ -102,7 +114,22 @@ define([
             },
 
             clickCloseDropdown: function(event, context) {
-                var $el = $(event.target);
+                var $el = $(event.target),
```

Comment:
Actually it is the target I want, not the currentTarget (the listener is on the page level and I am checking whether the user clicked on the dropdown button or not).